### DISCUSSION TITLE
The ability to copy the contents of a cell to the clipboard

### DISCUSSION THREAD

**Discussion by adithyabsk** [grafana/grafana#66313] [Dashboards, Panels & Visualizations]
Right now we have the ability to make a table cell a link, but being able to copy the contents of a table cell to the clipboard would unlock many use cases. Currently manually selecting, copying, and pasting specific cells of interest.

More generally, being able to run a javascript snippet on specific cells would be very useful.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is a very broad feature request, and I'm wondering if it wouldn't be better served by the ability to export the table as a CSV so it could be manipulated by a script outside of Grafana. Feel free to further define your use case in an [issue](https://github.com/grafana/grafana/issues) if you would like to request the feature.
